hybris/egl/platforms/wayland/eglplatform_wayland.cpp

```diff
@@ -211,7 +219,30 @@ extern "C" wl_buffer *waylandws_createWlBuffer(EGLDisplay dpy, EGLImageKHR image
 	    return NULL;
 	}
 	if (img->target == EGL_WAYLAND_BUFFER_WL) {
-		WaylandDisplay *wdpy = (WaylandDisplay *)hybris_egl_display_get_mapping(dpy);
+		/* load hybris's libEGL at runtime to call hybris_egl_display_get_mapping() */
```

Comment:
i think it would be nicer if this happened in a separate function

Reply:
makes sense, separated wrapper code